### DISCUSSION TITLE
Cisco Umbrella and Goguardian

### DISCUSSION THREAD

**Discussion by cooltoo37** [3kh0/ext-remover#570] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
does anybody know how to disable goguardian and cisco umbrella i tried point blank but it does not work well with my version it just keeps turning back on Version 102.0.5005.197 (Official Build) (64-bit) chrome dede shimmer does not work because i don't have extensions allowed block list and there is many proxies but its slow so can anybody help me disable goguardian and cisco umbrella

**Comment by srets** [comment, marked as answer]
is it turning on and off constantly

**Reply by cooltoo37** [reply]
yes

**Reply by srets** [reply]
thats supposed to happen

**Comment by CheezeKake77** [comment]
If you are on 102 I think ltbeef and ingot work still, correct me if I'm wrong

**Reply by cooltoo37** [reply]
no i am in like the lower versions so no its not gonna work

**Reply by cooltoo37** [reply]
102 earlier versions work not like the old ones
